airbyte-cdk/python/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.1.10
+Upd multiple token support: switch to list of tokens
+
+## 0.1.9
```

Comment:
do we need two versions for the same release? can we just make it a single version?

Reply:
Probably not. But both are released already